packages/mineral-ui-tokens/tokens/button/index.json

```diff
@@ -0,0 +1,11 @@
+{
+  "global": {
+    "type": "button",
+    "category": "button"
+  },
+  "props": [
+    { "name": "button_size_medium", "value": "35px" },
```

Comment:
medium => md
large => lg
jumbo => xl

Also, per the discussion yesterday, you can just make the values numbers, correct?
e.g. `"value": 35`